products/jbrowse-cli/src/commands/add-track.ts

```diff
@@ -205,14 +207,7 @@ export default class AddTrack extends JBrowseCommand {
       this.log(`Inferred default assembly name ${assemblyNames}`)
     }
 
-    let configObj = {}
-    if (config) {
-      try {
-        configObj = JSON.parse(config)
-      } catch (error) {
-        this.error('Could not parse provided JSON object')
-      }
-    }
```

Comment:
This one could use a suggestion, too, since otherwise it will just throw a generic JSON parsing error

Reply:
I added json-parse-better-errors. I think this can helkp users contextualize more or less automatically without custom try/catch. Popular package, 12M users